xds/src/main/java/io/grpc/xds/XdsInitializationException.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.xds;
+
+/**
+ * Throws when fail to bootstrap or initialize the XdsClient.
```

Comment:
Currently this is being used only for bootstrap errors. Unless you anticipate using this exception in "XdsClient initialization" cases how about calling this `BootstrapException` or `BootstrapLoadException`?

Reply:
Yes, it's also gonna be used by consumers of `BootstrapInfo`. For example, when you try to create the xDS channel with `BootstrapInfo` that contains a list of xDS server URIs and channel creds options, if none of the channel creds is supported, such an exception will be thrown.

Reply:
SG